CODE_OF_CONDUCT.md

```diff
@@ -1,15 +1,15 @@
-## Community Participation Guidelines
+# Community Participation Guidelines
 
-This repository is governed by Mozilla's code of conduct and etiquette guidelines.
-For more details, please read
-[Mozilla Community Participation Guidelines](https://www.mozilla.org/about/governance/policies/participation/).
+This repository is governed by Mozilla's code of conduct and etiquette guidelines. 
+For more details, please read the
+[Mozilla Community Participation Guidelines](https://www.mozilla.org/about/governance/policies/participation/). 
 
 ## How to Report
-For more information on how to report violations of the CPG, please read our '[How to Report](https://www.mozilla.org/en-US/about/governance/policies/participation/reporting/)' page.
+For more information on how to report violations of the Community Participation Guidelines, please read our '[How to Report](https://www.mozilla.org/about/governance/policies/participation/reporting/)' page.
 
+<!--
+## Project Specific Etiquette
 
-## Contributions
-
-For all other questions about contribution see our
-[contributing.md](https://github.com/mozilla/experimenter/blob/master/contributing.md)
-document.
+In some cases, there will be additional project etiquette i.e.: (https://bugzilla.mozilla.org/page.cgi?id=etiquette.html).
+Please update for your project.
+-->
```

Comment:
Ah crap you got trapped by the CODE_OF_CONDUCT mishap of 2019.  I think if you rebase this on master this should go away.

Reply:
Aww man, something funky went with the initial rebase...
Ended up duplicating the branch, hard resetting the original, pulled and cherry picked 🙈

Reply:
The dark side of the Git is a pathway to many abilities some consider to be unnatural.


![](https://i.pinimg.com/originals/9c/7f/1c/9c7f1ca55439422a6f7e98d487977da7.gif)